src/ctapipe_io_nectarcam/__init__.py

```diff
@@ -433,6 +435,15 @@ class NectarCAMEventSource(EventSource):
         help='If True, flat field and pedestal events are also calibrated.'
     ).tag(config=True)
 
+    def _correct_tel_id(self,tel_id,run):
```

Comment:
Make static method and move after init?

Reply:
As the only purpose is only to be applied to this class, what would be the purpose of making it a static method ? (I'm not an expert of python, so I don't know all the good practice and techniques)